evansmusomi/burger-builder
# Saves Customer Order on the Server

*What this does*
- Saves customer order info on Firebase
- Displays a spinner when the request is being processed

*How to verify*
- User can continue to check out and have their burger and customer info saved on the server
- User can see a spinner as their request to continue to checkout is being processed
